### DISCUSSION TITLE
Put resolution choice option.

### DISCUSSION THREAD

**Discussion by Jeff-2612** [skyline-emu/skyline#1035] [Ideas]
You can put a resolution choice menu, to have more performance in certain games.

**Comment by Crytonics** [comment]
Normally emulators have the option of 2x or 4x for higher resolution like AetherSX2. So you wont gain more fps with resolution.

**Reply by Crytonics** [reply]
Well you have docked and Handheld mode, Handheld is 720p so you can gain fps there.

**Comment by PixelyIon** [comment]
While we plan a resolution scaler, it is not short term as we're more focused on games rendering at all at the moment.

**Reply by Kermitsongs** [reply]
You are legends man
